src/plugins/intel_npu/src/utils/src/zero/CMakeLists.txt

```diff
@@ -2,26 +2,50 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-set(TARGET_NAME openvino_npu_zero_result_parser)
+set(TARGET_NAME openvino_npu_zero_utils)
 
 file(GLOB_RECURSE SOURCES *.cpp)
+source_group(TREE ${CMAKE_CURRENT_SOURCE_DIR} FILES ${SOURCES})
 
 add_library(${TARGET_NAME} STATIC ${SOURCES})
 set_target_properties(
   ${TARGET_NAME} PROPERTIES INTERPROCEDURAL_OPTIMIZATION_RELEASE ${ENABLE_LTO})
 ov_add_clang_format_target(${TARGET_NAME}_clang FOR_TARGETS ${TARGET_NAME})
 
-target_include_directories(
-  ${TARGET_NAME}
+add_library(openvino::npu_zero_utils ALIAS ${TARGET_NAME})
+set_target_properties(${TARGET_NAME} PROPERTIES EXPORT_NAME npu_zero_utils)
+
+target_include_directories(${TARGET_NAME}
```

Comment:
shall we change target_link_libraries and target_include_directories all be public? The target is exported, and it depends on ze_loader, but we only show header deps here

Reply:
done